smtsolver.js

```diff
@@ -44,7 +45,9 @@ function solve (query) {
     if (
       !solverOutput.startsWith('sat') &&
       !solverOutput.startsWith('unsat') &&
-      !solverOutput.startsWith('unknown')
+      !solverOutput.startsWith('unknown') &&
+      !solverOutput.startsWith('(error') &&
```

Comment:
What outputs `(`?

Reply:
@leonardoalt ^^

Reply:
ah, Eldarica outputs errors in some sort of smtlib sexpr, like `(error something failed lalala)` which is caught here